content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -131,9 +131,10 @@ As a cluster administrator, you can use a [PodSecurityPolicy](/docs/concepts/pol
 ### Generic ephemeral volumes
 
 {{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
 
 This feature requires the `GenericEphemeralVolume` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be
-enabled. Because this is an alpha feature, it is disabled by default.
+enabled. Because this is a beta feature, it is enabled by default.
 
 Generic ephemeral volumes are similar to `emptyDir` volumes, just more
```

Comment:
Similar to emptyDir in what way? I can only think of lifetime as the only similarity.

Reply:
They are also empty by default because they get provisioned anew for each pod startup. That they don't *have* to be empty falls under "more flexible"

Reply:
Changed into:
> Generic ephemeral volumes are similar to `emptyDir` volumes in the
> sense that they provide a per-pod directory for scratch data that is
> usually empty after provisioning. But they may also have additional
> features: